### PR TITLE
sketch: fix regex typo in rough.js

### PR DESCRIPTION
**Pull request by nekikara**
Thank you for an incredible diagram scripting tool.
I found the regex is different from that in [rough-stuff/rough](https://github.com/rough-stuff/rough/blob/master/src/generator.ts#L172).

### CODE REVIEW

**Review by alixander — CHANGES_REQUESTED** [review]
wow, nice, how did you find this? 

can you run ci? https://github.com/terrastruct/d2/blob/master/docs/CONTRIBUTING.md#ci
Just fetch the CI submodule and run `./make.sh`

File: d2renderers/d2sketch/rough.js [hunk 1]
```diff
@@ -1261,7 +1261,8 @@ class B {
       n = [];
     if (!t) return this._drawable("path", n, s);
     const i = (function (t, e) {
-      t = (t || "").replace(/\n/g, " ").replace(/(-\s)/g, "-").replace("/(ss)/g", " ");
+      // prettier-ignore
```
Comment (alixander) [inline d2renderers/d2sketch/rough.js:1264]: let's actually add this to https://github.com/terrastruct/d2/blob/master/.prettierignore
it's not made to be read anyway

**Comment by nekikara** [comment]
> wow, nice, how did you find this?

When I looked into the library to try solving issue #533, I noticed the difference :)

**Review by alixander — CHANGES_REQUESTED** [review]
sweet, looks like a render actually did change, though i can't see the visual difference.

can you run the test updates on go 1.18? (go 1.19 currently has an issue with breaking our ascii art.) i think some of the other tests are changing due to that.

**Review by alixander — APPROVED** [review]

**Comment by alixander** [comment]
@nekikara sorry just one more hoop, it's a one-time thing with having signed commits:

https://github.com/terrastruct/d2/pull/557#issuecomment-1367468730

**Comment by nekikara** [comment]
close by https://github.com/terrastruct/d2/pull/569

**Pull request closed without merging**